src/test/java/seedu/address/logic/commands/CommandTestUtil.java

```diff
@@ -194,4 +200,32 @@ public static void showPersonAtIndex(Model model, Index targetIndex) {
 
         assertEquals(1, model.getFilteredPersonList().size());
     }
+
+    /**
+     * Updates {@code model}'s filtered task list to show only the person at the given {@code targetIndex} in the
+     * {@code model}'s taskbook.
+     */
+    public static void showTaskAtIndex(Model model, Index targetIndex) {
+        assertTrue(targetIndex.getZeroBased() < model.getFilteredTaskList().size());
+
+        Task task = model.getFilteredTaskList().get(targetIndex.getZeroBased());
+        final String[] splitTaskLabel = task.getLabel().checkedLabel.split("\\s+");
+        model.updateFilteredTaskList(new TaskContainsKeywordsPredicate(Arrays.asList(splitTaskLabel[0])));
```

Comment:
Does this work when there's another task with the same label? Since we don't enforce unique labels (unlike the name in the client list)

Reply:
I'm not sure what your concern is; if there are multiple tasks with the same label then they will all be displayed, in the same way if you search the keyword Tan, even though our contact list enforces "uniqueness", the find command will display both Jane Tan and John Tan

Reply:
I agree, the find command works as expected. I was referring to the method itself, which shows only the task at the given index. I was wondering if it'd still display a single task if there were other tasks with the same label

Reply:
Ah I understand your concern now; and the answer is no.  If you look below, this is the reason for the changes to TypicalTasks. But this is the exact same way showPersonAtIndex works; they are built solely for tests, and how they work is based on the particular tests in which they're used. showPersonAtIndex would similarly fail if the model happened to have multiple people with the same first name. Hope this makes sense

Reply:
Yep that makes sense, I guess we just need to keep this in mind while writing tests for filtered lists